third_party/move/testing-infra/transactional-test-runner/tests/vm_test_harness/example.exp

```diff
@@ -3,19 +3,45 @@ processed 10 tasks
 task 3 'view'. lines 13-15:
 [No Resource Exists]
 
+task 4 'publish'. lines 17-35:
+Error: compilation errors:
+ error: duplicate declaration, item, or annotation
+   ┌─ TEMPFILE:4:11
+   │
+ 4 │ module A::N {
+   │           ^ Duplicate definition for module '(A=0x42)::N'
+   │
+   ┌─ TEMPFILE2:18:11
+   │
+18 │ module A::N {
+   │           - Module previously defined here, with '(A=0x42)::N'
+
+
+
 task 5 'run'. lines 37-37:
 Error: Function execution failed with VMError: {
-    major_status: ABORTED,
-    sub_status: Some(0),
-    location: 0x42::N,
+    major_status: FUNCTION_RESOLUTION_FAILURE,
+    sub_status: None,
+    location: undefined,
     indices: [],
-    offsets: [(FunctionDefinitionIndex(0), 1)],
+    offsets: [],
 }
 
+task 6 'run'. lines 39-45:
+Error: compilation errors:
+ error: no function named `N::give` found
+   ┌─ TEMPFILE3:42:9
+   │
+42 │         A::N::give(&s)
+   │         ^^^^^^^^^^^^^^
+
+
+
 task 7 'view'. lines 47-49:
```

Comment:
Could you check if these differences in output are acceptable and why? Seems like the v2 output was pre-existing.

Reply:
There are two "A::N" published in this test. The testing infra puts these two modules into two separate temp files and when publishing the latter one, the previous one is added as a dependency. Compiler V2 then generates an compilation error of duplicated modules while V1 does not. 
I commented out the first publish command and the updated exp file matches with the V1 version.